### DISCUSSION TITLE
avm-portfw has dependency FREETZ_AVM_VERSION_07_5X_MAX

### DISCUSSION THREAD

**Discussion by fraternl** [Freetz-NG/freetz-ng#964] [Unerfüllte Wünsche]
I tried a 7.80 compile for a 7590AX and found out that a dependency of one of my packages for **avm-portfw** was in conflict with the dependency of **avm-portfw** itself.
in pkgs/avm-portfw/Config.in it has
`
depends on FREETZ_AVM_VERSION_07_5X_MAX`

This means the package **avm-portfw** can't be selected in 7.80 and I need it.
Are major versions above 7.50 excluded intentionally?
If so, will there be an effort to enable it for that version?

I have only 1 7590AX and it can run 7.5 just fine if needed.
I do hope I will be able to shift to 7.8 when it becomes available for the 7490 and 7590 which I have plenty.

**Comment by fda77** [comment]
"people" "reported" it "does not walk" anymore with fos 7.8 ("geht net")
I can not say if this is true as i have no deivce to test by myself as i have not a avm device which supports such a new fos version. An offline-test without device is not possible

 - You could remove the "FREETZ_AVM_VERSION_07_5X_MAX" limitation and test if it work
 - check how ar7 was changed
 - and/or why it does not was able to change the file
 - maybe avm uses a complete other way to open port
 - find how avm does it
 - fix the freetz package to support it again

**Reply by Bodenseematze** [reply]
@fraternl

I would really be interested in how AVM does store local port forwarding in 7.8

It would really be nice if you would be able to do some tests...
First go to the AVM web interface enable the HTTPS access from internet and set a special (remarkable) port for it (e.g. 7432) - in German FOS you can find that setting at "Internet/Freigaben/FRITZ!Box Dienste" - I don't know how it's called in English version or if it has changed in 7.8...
After doing that (and saving it of course) please open either a terminal session at your box and store your /var/flash/ar7.cfg and transfer it to your PC or use the Freetz Rudi-Shell to directly transfer the file to your PC.
(after that you can of course go again to the AVM web interface and remove / change that HTTPS access again...)

Open the ar7.cfg with an text editor which is able to handle Unix lineendings and search for the above special port (7432) and post all sections of the file where it occurs here...

(and/or you remove / xxx out sensitive data in your ar7.cfg and post the whole file here)

**Comment by fraternl** [comment]
_**You could remove the "FREETZ_AVM_VERSION_07_5X_MAX" limitation and test if it work**_

I just removed the `"FREETZ_AVM_VERSION_07_5X_MAX"` and compiled a 7590AX 7.8 firmware.
Then I wanted to flash the 7590AX I bought especially for testing....   and it's gone.

I think it went to a customer in a batch of 10 7590's I prepared with Freetz.
So, first I need to get it back before I can do any testing.

**Comment by fraternl** [comment]
I now know where the router is.
It went to site where I'm testing a setup and can access it remotely.

I just tried to flash the router using the Freetz-interface and it came back with an error.
Is this to be expected?

![image](https://github.com/Freetz-NG/freetz-ng/assets/8692856/32642c61-a327-4cc4-8be5-5c59336fadeb)

![image](https://github.com/Freetz-NG/freetz-ng/assets/8692856/742f0205-112d-4c9b-aacd-e72a82a0ede2)

**Reply by fda77** [reply]
I have no idea + no hardware which could run such a new FOS.
Please repost into another discussion, its OT here

**Comment by fraternl** [comment]
It turned out that the inability to flash the router through the Freetz interface is a bug of 7.80
The 7590AX I'm able to access is accessed remotely through a tunnel and was already running 7.8, but one I flashed several months ago.
It also had a 7.5 firmware as a reserve on it, so I toggled it and then flashed a new 7.8 firmware.
This was successful.
There is still something wrong with flashing a firmware using the freetz webinterface in 7.8, but it's unrelated.
I tried to flash the same firmware again whilst already running the 7.8 firmware and it failed again.

So now it's running **yesterday's** version of 7.8 with avm-portforwarding enabled.
If I fill in a port or 2 ports there, it returns with an empty field.
It also doesn't open any ports

**So, merely removing the _"FREETZ_AVM_VERSION_07_5X_MAX"_ limitation doesn't work.**

I can't really be testing more as I can't afford to lose that router there because I'm using it to test a routed subnet with a Mikrotik.

**Reply by fda77** [reply]
So in short you are demanding someone (aka random reveloper guy) to pop up and buys a device which can run the software version you are using and adds a feature you want to sell to you customers?
Me toi, we have to wait

**Comment by fraternl** [comment]
> So in short you are demanding someone

Can you point me to the line where I "demand" anything?

Maybe you should assume less or at least not "accuse" people of having certain intentions.
I'm providing feedback here. If it's not welcome I have totally no problem to refrain from giving feedback.

I bought this 7590AX to see if it can, in the near future, be a replacement for the 7590's.
I have all the time in the world and if I have to wait another year before 7.8 can be used I have totally no problem with that.

I'm not "selling" anything.
The only one profiting from all this is AVM.
I even think there soon comes a time we don't use those routers anymore as other routers are "open" without tinkering.
We're already using them for most scenario's.

**Reply by fda77** [reply]
> I think it went to a customer in a batch of 10 7590's I prepared with Freetz.

You sell 10 devices to "customer" which dont pay, but you give them money?

Why do you post here? The problem is known and you dont want to help. I see only 1 reason

But this is OT

**Comment by fraternl** [comment]
Why don't you remove your own comment as well?
It's untrue and offensive

**Comment by fraternl** [comment]
and off-topic

**Comment by fda77** [comment]
bye
